src/ploomber/sources/notebooksource.py

```diff
@@ -419,8 +419,7 @@ def _post_init_validation(self, value):
                 msg += ('. Add a cell at the top and tag it as "parameters". '
                         'Go to the next URL for '
                         'details: https://ploomber.io/s/params')
-
-            raise MissingParametersCellError(msg)
+            click.secho(msg)
```

Comment:
since we are adding the cell anyway, we no longer need this message

Reply:
Do we want to remove this whole method (`_post_init_validation`)?

Reply:
keep the method but leave it empty (add a `pass` to the body), as other parts of the framework call it